src/test/data/script_tests.json

```diff
@@ -2125,5 +2125,88 @@
 ["0", "CHECKSEQUENCEVERIFY", "CHECKSEQUENCEVERIFY", "UNSATISFIED_LOCKTIME", "CSV fails if stack top bit 1 << 31 is set and the tx version < 2"],
 ["4294967296", "CHECKSEQUENCEVERIFY", "CHECKSEQUENCEVERIFY", "UNSATISFIED_LOCKTIME",
   "CSV fails if stack top bit 1 << 31 is not set, and tx version < 2"],
-["The End"]
+
+["MINIMALIF tests"],
```

Comment:
I'm not sure what should(not) be tested. I want to show that the new rules are not applied to non segwit scripts by accident